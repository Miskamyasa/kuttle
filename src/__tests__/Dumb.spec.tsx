import {render, screen} from "@testing-library/react"


describe("Dumb test", () => {
  it("renders div element", () => {
    const text = "Some Text"
    render(<div>{text}</div>)
    expect(screen.getByText(text)).toBeInTheDocument()
    expect(screen.getByText(text)).toMatchSnapshot()
  })
})
