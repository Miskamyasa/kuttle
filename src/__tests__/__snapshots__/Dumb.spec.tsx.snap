// Vitest Snapshot v1

exports[`Dumb test > renders div element 1`] = `
<div>
  Some Text
</div>
`;
